gamemodes/cinema/gamemode/player_class/player_lobby.lua

```diff
@@ -0,0 +1,97 @@
+AddCSLuaFile()
+DEFINE_BASECLASS( "player_default" )
```

Comment:
isn't this file from the other repo

Reply:
It's a modified version of the stock cinema one

Reply:
I see, I already have one of these files in my private repo, so, please just remove it from this PR and i'll incorporate the changes from here manually